terraform/modules/cloudfoundry/elb_apps.tf

```diff
@@ -34,7 +34,7 @@ resource "aws_lb_listener" "cf_apps" {
   load_balancer_arn = aws_lb.cf_apps.arn
   port              = "443"
   protocol          = "HTTPS"
-  ssl_policy        = "ELBSecurityPolicy-TLS-1-2-2017-01"
+  ssl_policy        = "ELBSecurityPolicy-TLS13-1-0-2021-06"
```

Comment:
I *think* this should be fine because the documentation says:

> *For TLS listeners, we recommend using the ELBSecurityPolicy-TLS13-1-2-2021-06 security policy. This is the default policy for listeners created using the AWS Management Console. This security policy includes TLS 1.3, which is optimized for security and performance, and is backward compatible with TLS 1.2.

https://docs.aws.amazon.com/elasticloadbalancing/latest/network/create-tls-listener.html#:~:text=*For%20TLS%20listeners,with%20TLS%201.2.

So this policy should still support TLS 1.2 and thus not break any existing clients. But I'd like to have someone confirm before we approve this.

Reply:
Can we change the PR to use `TLS13-1-2-Ext1-2021-06` as the security policy instead? It has support for TLS1.3 but full backwards compatibility with our existing ciphers.

You can see the comparison spreadsheet we made here: https://docs.google.com/spreadsheets/d/1bX1ywhFafAfOPlo8dCNqWw4J5E8chXiQFbtp__-AbQ0/edit#gid=0